P4L_Epoch.md

```diff
@@ -183,10 +183,8 @@ No more changes needed in this file. RESTORE TO THE NORMAL REMOVES.SQF
 ALL THREE OF THESE FILES NEED THE SAME EDIT, MAKE SURE YOU DO ALL FILES!!!!<br>
 Find
 	
-	_friendlies		= player getVariable ["friendlyTo",[]];
-	// check if friendly to owner
-	if(_ownerID in _friendlies) then {
-		_canBuildOnPlot = true;
+	if ((_isowner) || (_isfriendly)) then {
```

Comment:
Are you 100% sure all these variables exists without adding it ourselfs. 

I didn't check it, but if you say so i'll believe you.